spec/models/search/tag_query_spec.rb

```diff
@@ -194,12 +194,62 @@
 
     it "allows you to sort by Uses" do
       q = TagQuery.new(sort_column: "uses")
-      expect(q.generated_query[:sort]).to eq([{ "uses" => { order: "desc" } }, { id: { order: "desc" } }])
+      expect(q.generated_query[:sort]).to eq([{ "uses" => { order: "desc" } }, { "name.keyword" => { order: "asc" } }, { id: { order: "desc" } }])
     end
 
     it "allows you to sort by Uses in ascending order" do
       q = TagQuery.new(sort_column: "uses", sort_direction: "asc")
-      expect(q.generated_query[:sort]).to eq([{ "uses" => { order: "asc" } }, { id: { order: "asc" } }])
+      expect(q.generated_query[:sort]).to eq([{ "uses" => { order: "asc" } }, { "name.keyword" => { order: "asc" } }, { id: { order: "asc" } }])
+    end
+  end
+
+  describe "to_wrangle" do
```

Comment:
This test should have the tag `tag_search: true` on it, to reset the search index.

https://github.com/otwcode/otwarchive/blob/e211c6bb9ad792e344aacb629020c3bc92576b0a/spec/spec_helper.rb#L106-L112